airbyte-integrations/bases/base-normalization/dbt-project-template/macros/cross_db_utils/json_operations.sql

```diff
@@ -249,5 +270,5 @@
 
 # https://cloud.google.com/bigquery/docs/reference/standard-sql/json_functions#json_extract_string_array
 {% macro bigquery__json_extract_string_array(json_column, json_path_list, normalized_json_path) -%}
-    json_extract_string_array({{ json_column }}, {{ format_json_path(normalized_json_path) }})
+    json_query_array({{ json_column }}, {{ bigquery_new_format_json_path(normalized_json_path) }})
```

Comment:
I think you need to use equivalent counterpart:

OLD `JSON_EXTRACT_ARRAY` -> NEW `JSON_QUERY_ARRAY` - for array of objects
OLD `JSON_EXTRACT_STRING_ARRAY` -> NEW `JSON_VALUE_ARRAY` - for array of scalar values